server.js

```diff
@@ -16,7 +18,11 @@ server.engine('hbs', hbs({
   defaultLayout: 'main',
   layoutDir: path.join(__dirname, './', 'views/layouts'),
   partialsDir: path.join(__dirname, './', 'views/partials'),
-  extname: 'hbs'
+  extname: 'hbs',
+  helpers: {
+    prettyDate: (date) => dateFormat(date, 'dddd d - mmm - yyyy'),
+    prettyTime: (time) => date.format(new Date(time), 'HH:mm A', true)
```

Comment:
(also, you use `HH` which is 24-hour clock, so you don't need the `A` which does `am` or `pm`